kernel/univ.ml

```diff
@@ -963,6 +963,15 @@ struct
 
 end
 
+type 'a univ_abstracted = {
+  univ_abstracted : 'a;
+  bound_univs : Names.Name.t array
+}
```

Comment:
By the way, why a list of names rather than a `AUContext.t`? Even if you explain explicitly later that the bindings are à la de Bruijn, it is a bit misleading to see names, as if the universes were bound by names.

Reply:
I support reusing the `AUContext.t` type. It is more informative, and that information might end up being needed, or allowing to thread data in a uniform way.